tools/st2ctl

```diff
@@ -6,7 +6,7 @@ LOGFILE="/tmp/st2_startup.log"
 WEBUI_LOG_FILE="/var/log/st2/st2web.log"
 COMPONENTS="st2actionrunner st2api st2auth st2sensorcontainer st2rulesengine mistral st2web st2resultstracker st2notifier"
 STANCONF="/etc/st2/st2.conf"
-PYTHON=`which python`
```

Comment:
You could always check exit code != 0 and either bail out or switch to `which python`.